.github/workflows/run_tests.yml

```diff
@@ -8,15 +8,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.10', '3.12']
+        python-version: ['3.9', '3.11', '3.13']
         # currently there is conflict between tf, oneflow and paddle in protobuf versions.
         # cupy is not tested because it demands gpu
         # oneflow testing is dropped, see details at https://github.com/Oneflow-Inc/oneflow/issues/10340
         # paddle was switched off because of divergence with numpy in py3.10, paddle==2.6.1
         # The last pytensor release that supports python 3.8 doesn't include einsum, so we skip that combination.
```

Comment:
Update to mention the opposite, that last pytensor does not yet support python3.13?

Reply:
will do if end with 3.13 working.  Figuring out blockers right now.